5.least_common_multiply.py

```diff
@@ -0,0 +1,11 @@
+def divisibleSumPairs(n, k, ar):
+
+    sums = 0
+    for i in range(n):
+        for j in range(n):
+            if i < j and (ar[i] + ar[j]) % k == 0:
+                sums += 1
+    return sums
+    #return sum(1 for i in range(n) for j in range(n) if i < j and (ar[i]+ar[j])%k == 0)
+
+print(divisibleSumPairs(6, 3, [1, 3, 2, 6, 1, 2]))

```

Comment:
In the question its about the least common multiply. For example if our numbers are 12 and 20, its result should be (12*20)/4(greatest common divisor of the 12 and 20 which is 4)=60. you should try to make a function like this. 

I only understand here for your code; you couldn't get the point that this question mean.

Reply:
Abi, bu sorunun cevabi hacker rankte cozdugum baska bir sorunun cevabi ile karismis. Yanlislikla taslak olarak kullandigim sayfa yerine sorunun cozumunu ustune yapistirmisim. Bu sorunun asil cozumu bende baska bir dosyada mevcut. Dilerseniz iletebilirim.

Reply:
send here, please with using git. If you can't, paste it below this comment.

Reply:
def least_common_multiply(num1,num2):

    min_multiply = []

    divisors1 = ([i for i in range(1, num1+1) if num1 % i == 0])  # We find the divisors of the first number.
    divisors2 = ([j for j in range(1, num2+1) if num2 % j == 0])  # We find the divisors of the second number.

    # We find the all possible multiplies of each number by multiplying the numbers with the other number's divisors.
    multiply1 = [m*num1 for m in divisors2]
    multiply2 = [n*num2 for n in divisors1]

    return min(set(multiply1).intersection(set(multiply2)))
    # We take the intersection of two lists as sets and use min function to find the least common multiply.


print(least_common_multiply(3, 4))